backend/infrahub/task_manager/task.py

```diff
@@ -84,8 +86,24 @@ async def _get_related_nodes(cls, db: InfrahubDatabase, flows: list[FlowRun]) ->
 
     @classmethod
     async def _get_logs(cls, client: PrefectClient, flow_ids: list[UUID]) -> FlowLogs:
+        offset = 0
+        all_logs = []
+
+        while True and offset < NB_LOGS_LIMIT:
+            # Retrieve logs with the current offset, without specifying a limit
+            logs_batch = await client.read_logs(
```

Comment:
unless I'm reading it wrong, it feels like we'll always make 2 calls to `client.read_logs` with this methods even if the number of logs is small

I think we should set the limit when we query the logs, this way we'll know right away if we need to do another call or not and we won't need a second call to return empty